doc/apache2/Readme.md

```diff
@@ -70,7 +70,7 @@ Example config for Apache 2.4 in prefork mode:
         # Makes it possible to place your favicon at the root `web` directory in your eZ instance.
         # It will then be served directly.
         RewriteRule ^/favicon\.ico - [L]
-        RewriteRule ^/robots\.txt - [L]
+        # RewriteRule ^/robots\.txt - [L]
```

Comment:
I think we'll have some merge conflicts during the upgrades if we comment this line. 
Do we have a better option?